java/src/jmri/jmrix/can/cbus/node/CbusNodeConstants.java

```diff
@@ -346,10 +346,12 @@ private static Map<Integer, String> createType44Map() {
         result.put(3, "CANSPROG"); // NOI18N
         result.put(4, "SBOOST"); // NOI18N
         result.put(5, "Unsupported"); // NOI18N
-        result.put(6, "CANISB"); // NOI18N
-        result.put(7, "CANCBUSIO"); // NOI18N
-        result.put(8, "CANSERVOIO"); // NOI18N
-        result.put(9, "CANSOLIO"); // NOI18N
+
+        result.put(8, "CANSOLNOID"); // NOI18N matches MERG CANSOL
+        result.put(50, "CANSERVOIO"); // NOI18N matches MERG canmio-svo
```

Comment:
These comments look a bit strange. I suggest something like:
// NOI18N. Matches MERG CANSOL

Reply:
Or maybe:
`// matches MERG CANSOL`

The tool we used to check NOI18N doesn't work anymore so we have removed it. So the comment NOI18N isn't needed anymore. The reason to why we removed this tool was that Java and JMRI has evolved so that it's too difficult to validate I18N in the Java code today. The tool we used isn't intelligent enough to handle our current code base.

Reply:
Thanks, Done.